content/06.methods.md

```diff
@@ -188,56 +179,57 @@ CNVs called significant by Control-FREEC (p-value < 0.01) and Manta calls that p
-We also provided Arriba with strandedness information for stranded samples, or we set it to auto-detection for poly-A samples.
-We used [FusionAnnotator](https://github.com/FusionAnnotator/FusionAnnotator) on Arriba fusion calls in order to harmonize annotations with those of STAR-Fusion.
-The RNA expression and fusion workflows can be found in the [OpenPBTA workflows GitHub repository](https://github.com/d3b-center/OpenPBTA-workflows/blob/master/cwl/kfdrc_RNAseq_workflow.cwl).
-The FusionAnnotator workflow can be found in the [D3b GitHub repository](https://github.com/d3b-center/FusionAnnotator).
+We set up Arriba v1.1.0 [@doi:10.1101/gr.257246.119] and STAR-Fusion 1.5.0 [@doi:10.1101/120295] fusion detection tools using CWL on CAVATICA.
+For both of these tools, we used aligned BAM and chimeric SAM files from STAR as inputs and `GRCh38_gencode_v27` GTF for gene annotation.
+We ran STAR-Fusion with default parameters and annotated all fusion calls with `GRCh38_v27_CTAT_lib_Feb092018.plug-n-play.tar.gz` provided in the STAR-fusion release.
+For Arriba, we used a blacklist file `blacklist_hg38_GRCh38_2018-11-04.tsv.gz` from the Arriba release to remove recurrent fusion artifacts and transcripts present in healthy tissue.
+We provided Arriba with strandedness information for stranded samples, or we set it to auto-detection for poly-A samples.
+We used [FusionAnnotator](https://github.com/FusionAnnotator/FusionAnnotator) on Arriba fusion calls to harmonize annotations with those of STAR-Fusion.
+The RNA expression and fusion workflows can be found in the [D3b GitHub repository](https://github.com/d3b-center/OpenPBTA-workflows/blob/master/cwl/kfdrc_RNAseq_workflow.cwl).
+The FusionAnnotator workflow can also be found in the [D3b GitHub repository](https://github.com/d3b-center/FusionAnnotator).
 
 ### QUANTIFICATION AND STATISTICAL ANALYSIS
 
 ##### Recurrently mutated genes and co-occurrence of gene mutations
 
-Using the consensus SNV calls, we identified genes that were recurrently mutated in the cohort, including nonsynonymous mutations with a variant allele frequency greater than 5% among the set of independent samples.
-The set of nonsynonymous mutations was determined using ENSEMBL Variant Effect Predictor [@doi:10/gdz75c] annotations, including High and Moderate consequence types as defined in `maftools v. 2.2.10` [@doi:10.1101/gr.239244.118].
+Using the consensus SNV calls, we identified genes that were recurrently mutated in the OpenPBTA cohort, including nonsynonymous mutations with a VAF > 5% among the set of independent samples.
+The set of nonsynonymous mutations was determined using ENSEMBL Variant Effect Predictor [@doi:10/gdz75c] annotations, including "High" and "Moderate" consequence types as defined in `maftools v. 2.2.10` [@doi:10.1101/gr.239244.118].
```

Comment:
We haven't been using code style elsewhere. Also suggested  `v. ` --> `v`, your call

```suggestion
The set of nonsynonymous mutations was determined using ENSEMBL Variant Effect Predictor [@doi:10/gdz75c] annotations, including "High" and "Moderate" consequence types as defined in maftools v2.2.10 [@doi:10.1101/gr.239244.118].
```